### DISCUSSION TITLE
No matching manifest for Linux/arm/v7 in the manifest list entries

### DISCUSSION THREAD

**Discussion by mgrewe17** [koush/scrypted#1310] [Q&A]
Hi All,

I have an older Hoobs (Banana Pi MP5).  I had Hoobs running in parallel with Scrypted.  Scrypted prompted me to run an update, which I did and then it never restarted a week ago.  Therefore, I flashed everything (Hoobs and Scrypted).  However, when I go to run the install for Scrypted I get the error in the title above.  FYI, I am upgrading the Hoobs Debian bullseye before running the Scrypted linux install.  I know the error is a docker error, but can't find a solution in any forum.  Any ideas?

There is a similar post on Reddit Forum (link below)
https://www.reddit.com/r/Scrypted/comments/18wlga5/no_matching_manifest_for_linuxarmv7_in_the/

I am still learning, so please be detailed in steps if anyone has any for me to help me out and to get back up and running.

**Comment by bjia56** [comment]
32 bit Linux systems are no longer supported by Scrypted. I don't know if Hoobs offers 64 bit images, but I think the Banana Pi you have should support a different 64 bit Linux image. I recommend checking out the official images supported Banana Pi and selecting a 64 bit version, then getting that running on the hardware, then installing Hoobs and Scrypted with their respective install docs.

**Reply by mgrewe17** [reply]
It is my understand and research that a Banana Pi MP5 is 32 bit only due to hardware.  If Scrypted no longer supports 32 bit hardware, Is there anyway to get an old version of Scrypted to reinstall?  I originally installed Scrypted in August of 2022 and everything worked.  I am more than happy to go back to that version for now.
